project_sla/test/project_sla.yml

```diff
@@ -35,7 +35,7 @@
   There should be Service Level info generated on the Issue
 -
   !assert {model: project.issue, id: issue1, string: Issue should have calculated service levels}:
-    - len(sla_control_ids) == 2
+    - len(sla_control_ids) == 1
```

Comment:
Look at [8.0 version of tests](https://github.com/OCA/project-service/blob/8.0/project_sla/test/project_sla.yml#L39)